app/controllers/patients_controller.rb

```diff
@@ -30,7 +30,7 @@ def show
   def new
     redirect_to(root_url) && return unless current_user.can_create_patient?
 
-    @patient = Patient.new(jurisdiction_id: current_user.jurisdiction_id)
+    @patient = Patient.new(jurisdiction_id: current_user.jurisdiction_id, isolation: params.permit(:isolation)[:isolation] == 'true')
```

Comment:
should this `isolation` param also be added to `new_group_member`?

Reply:
No need, `isolation` gets copied over by the `group_member_subset` in the `new_group_member` method.